### ISSUE TITLE
--qual option not working? 

### ISSUE THREAD

**Issue by dehui333** [HKU-BAL/Clair3#285]
Hi, 

I was trying with lower --qual thresholds but the change doesn't seem to be applied? 

My command:
```
run_clair3.sh --bam_fn=${BAM} --ref_fn=${REF} --bed_fn=${BED} --threads=$THREADS --qual='1' --platform='ont' --model_path=${MODEL_PREFIX} --output=${OUTPUT_DIR}
``` 

A line from merge_output.vcf:
```
chr19   85784   .       T       TA      1.57    LowQual F       GT:GQ:DP:AD:AF  0/1:1:138:79,59:0.4275
``` 

Perhaps I am doing something wrongly? Assistance would be appreciated, thanks.

**Comment by zhengzhenxian** [comment]
@dehui333 The issue was fixed in v1.0.6, pls kindly try our latest docker image.

**Comment by dehui333** [comment]
Thanks!
